viewing_party/party.py

```diff
@@ -0,0 +1,183 @@
+#Wave 1 first 4 tests
+def create_movie(movie_title, genre, rating):
+
+    if movie_title and genre and rating:
+#creating new_movie dictionary
+        new_movie = {
+            "title": movie_title,
+            "genre": genre,
+            "rating": rating
+        }
+        return new_movie
+
+    else:
+        return None
+
+#Wave 1 Tests 5 6 
+def add_to_watched(user_data, movie):
+    #adding movies to the watched list
+    user_data["watched"].append(movie)
+    return user_data
+
+#Wave 1 Tests 7 8
+def add_to_watchlist(user_data, movie):
+    #adding movies to watchlist
+    user_data["watchlist"].append(movie)
+    return user_data
+
+#wave 1 Tests 9 10 11
+def watch_movie(user_data, title):
+    
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            watched_dict = movie
+            break
+    else:
+        return user_data 
+    user_data["watched"].append(watched_dict)
+    user_data["watchlist"].remove(watched_dict)
+
+    return user_data
```

Comment:
This works, but the flow is a little awkward.  I would recommend restructuring for readability:
```suggestion
    for movie in user_data["watchlist"]:
        if movie["title"] == title:
            watched_dict = movie
            user_data["watched"].append(watched_dict)
            user_data["watchlist"].remove(watched_dict)
    
    return user_data
```